frontend/packages/db/schema/schema.sql

```diff
@@ -1538,6 +1607,11 @@ ALTER TABLE ONLY "public"."building_schemas"
 
 
 
+ALTER TABLE ONLY "public"."db_engines"
+    ADD CONSTRAINT "db_engines_project_id_fkey" FOREIGN KEY ("project_id") REFERENCES "public"."projects"("id") ON UPDATE CASCADE ON DELETE RESTRICT;
+
+
+
```

Comment:
I was wondering if I should set `projects`:`db_engines` = 1:1, but I think I need to set `github_repositories` has_many `projects` to do so, so I'm going to proceed as `projects`:`db_engines` = 1:many